config/src/config/indexer_table_info_config.rs

```diff
@@ -4,8 +4,8 @@
 use serde::{Deserialize, Serialize};
 
 // Useful defaults
-pub const DEFAULT_PARSER_TASK_COUNT: u16 = 20;
-pub const DEFAULT_PARSER_BATCH_SIZE: u16 = 1000;
+pub const DEFAULT_PARSER_TASK_COUNT: u16 = 10;
+pub const DEFAULT_PARSER_BATCH_SIZE: u16 = 100;
```

Comment:
why?

Reply:
we had better perf with lower batch and thread count?

Reply:
More thread lol. I'd keep these as is and let @larry-aptos add the dynamic optimization we applied to that side. We also need a whole lot of testing.